repl/src/test/scala/com/cloudera/livy/repl/PythonInterpreterSpec.scala

```diff
@@ -150,6 +149,7 @@ class PythonInterpreterSpec extends BaseInterpreterSpec {
       "name 'x' is not defined",
       List(
         "Traceback (most recent call last):\n",
+        "  File \"<stdin>\", line 1, in <module>\n",
```

Comment:
Sorry, I was going to clarify. The stack trace is ok, I meant the frames mentioning `<stdin>` and line numbers.